android/src/main/kotlin/live/hms/hmssdk_flutter/HmssdkFlutterPlugin.kt

```diff
@@ -1026,4 +1032,38 @@ class HmssdkFlutterPlugin : FlutterPlugin, MethodCallHandler, ActivityAware,
         val name = call.argument<String>("name");
         hmssdk.changeName(name=name!!, hmsActionResultListener = this.actionListener);
     }
+
+    private fun hlsStreaming(call: MethodCall) {
+        val meetingUrl = call.argument<String>("meeting_url")
+        val meetingUrlVariant1 = HMSHLSMeetingURLVariant(
+            meetingUrl = meetingUrl!!,
+            metadata = "tag for reference"
```

Comment:
why metadata is being populated from SDK? A value for metadata should come from client apps. Should be nil otherwise.